src/handler.js

```diff
@@ -149,7 +158,8 @@ export default class Handler {
             uri: this.endpoint,
             //json: true,
             headers: {
-                'reb-auth': this.authorization
+                'reb-auth': this.authorization,
+                'reb-api-consumer': `RebillySDK/JS-Token ${version}`
```

Comment:
I think it should be `Reb-Api-Consumer` (case)

Reply:
Based on RFC all header names are case insensitive so it wouldn't make any difference.

Reply:
Oh, okay. I'm wrong then.